microbench/src/test/java/io/netty/microbench/http2/Http2FrameWriterBenchmark.java

```diff
@@ -127,11 +130,11 @@ public EnvironmentParameters params() {
     @Setup(Level.Trial)
     public void setup() {
         switch (environmentType) {
-        case EMBEDDED_POOLED:
-        case EMBEDDED_UNPOOLED:
+          case EMBEDDED_POOLED:
+          case EMBEDDED_UNPOOLED:
             environment = boostrapEmbeddedEnv(environmentType);
             break;
-        default:
+          default:
```

Comment:
We don't use 2-space indentation.  I'm interested in utilizing our screen real-estate though. If you think it's a good idea to use 2-space indentation, please file an issue with the 'discussion' label.